package/python-kiwi-spec-template

```diff
@@ -29,7 +29,7 @@
 %endif
 
 %if %{undefined python3_sitelib}
-%global python3_sitelib %(%{__python3} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
+%global python3_sitelib %(%{__python3} -c "import sysconfig; print(sysconfig.get_path('platlib'))")
```

Comment:
Does this module exist on all Debian/Ubuntu versions we build against right now?

Reply:
in standard python for 3.9 and older:
* https://docs.python.org/fr/3.9/library/sysconfig.html

as we set a constraint to python >= 3.9 I believe this change should be safe